### PR TITLE
Update bahdanau2014neural_2.md

### PR DESCRIPTION
**Pull request by ayushchauhan**

### CODE REVIEW

File: _summaries/bahdanau2014neural_2.md [hunk 1]
```diff
@@ -1,19 +1,70 @@
...
+
+As earlier, the encoding vector(s) are computed from the hidden states generated by the encoder. However, here the hidden states are weighted by the parameters of an attention layer $$\alpha_{ij}$$:
+
+$$c_{i} = \sum_{j=1}^{T_x} \alpha_{i j} h_{j} \qquad \boldsymbol{\alpha}_i = softmax(\mathbf{e}_{i}) \qquad e_{ij} = a(s_{i-1},h_{j})$$
+
+where the last equation represents the parameters of an _alignment model_ that learns to assign an importance to inputs around $$j$$ for the output at $$i$$. This model $$a(\cdot)$$ is jointly trained with the other parts of the encoder-decoder architecture. This allows the model to search for positions in the input sequence with the most relevant information (captured in the context-vectors) for predicting next word.
+
+
+
+## Benefits of proposed model
```
Comment (slycane9) [inline _summaries/bahdanau2014neural_2.md:39]: Perhaps reference some of the qualitative analysis around the words alignment or long sentence translations here.

File: _summaries/bahdanau2014neural_2.md [hunk 2]
```diff
@@ -1,19 +1,70 @@
...
+<img width="500" alt="bahdanau2014neural_2c" src="bahdanau2014neural_2c.png">
+
+
+The table above compares the performance of the models on the test set of 3003 sentences as measured by the **BLEU score**. The last column corresponds to testing on sentences which do not contain any words not seen during training. RNNsearch-50* model is trained for much longer.
+
+* The translation model with attention clearly outperforms the baseline model by a substantial margin. 
+* In fact, the performance of the RNNsearch-30 is better than RNNencdec-50 which shows the strength of RNNsearch in encoding long sequences.
+* In the _No UNK_ case, the performance of the attention-based NMT model is also comparable with a conventional phrase-based machine translator (Moses) given that Moses uses an additional monolingual corpus of 418M words. 
+* The plot illustrates the superiority of RNNsearch as the sequence length increases. The performance of the baseline model deteriorates but RNNsearch-50 is stable even beyond the length of 50.
+
```
Comment (slycane9) [inline _summaries/bahdanau2014neural_2.md:65]: This is a very well done review so I struggled to find much wrong with it.  Perhaps condense the explanation of the decoder slightly and explain more the prior implementations of encoder-decoder networks.

**Review by slycane9 — COMMENTED** [review]
Well done review, just maybe reference qualitative results and prior encoder-decoder implementation slightly more.

File: _summaries/bahdanau2014neural_2.md (hunk 1 above)
Comment (ayushchauhan) [reply _summaries/bahdanau2014neural_2.md:39]: Sure, I can do that in brief. I was just worried about the length of the summary but I can refer it in a sentence. Thanks!

File: _summaries/bahdanau2014neural_2.md (hunk 2 above)
Comment (ayushchauhan) [reply _summaries/bahdanau2014neural_2.md:65]: Thanks! The prior implementations are the same as the 2 papers previous to this one for this lecture. They are going to be summarized and discussed there, so I didn't go too deep into that and explained the proposed architecture in detail.

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**